### PR TITLE
Halloween 2020

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
Adds the community event Halloween costumes.

## Why It's Good For The Game
Something something event.

## Changelog
:cl: Guillaume Prata/ The Community 
add: Added multiple costumes
del: Removed some broken costumes
tweak: tweaked the Halloween event from last year
fix: fixed some costumes, Rip Shrek
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
Yes

**Comment by SgtHunk** [comment]
Looks good!

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 1]
```diff
@@ -0,0 +1,311 @@
+//Hotdog
+/obj/item/clothing/suit/hotdog
+	name = "Hotdog"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:3]: probably should be lowercase

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "Hot Dawg."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "hotdog"
+
+/obj/item/clothing/head/hot_head
+	name = "Hotdog hat"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:10]: same here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+/obj/item/storage/box/halloween/edition_19/hotdog
+	theme_name = "2019's Hotdog"
+
+/obj/item/storage/box/halloween/edition_19/hotdog/PopulateContents()
+	new /obj/item/clothing/suit/hotdog(src)
+	new /obj/item/clothing/head/hot_head(src)
+
+//---Ketchup
+/obj/item/clothing/suit/ketchup
+	name = "Ketchup"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:27]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "A soft plush ketchup bottle."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "ketchup"
+
+/obj/item/clothing/head/ketchup_head
+	name = "Ketchup hat"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:34]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+/obj/item/storage/box/halloween/edition_19/ketchup
+	theme_name = "2019's Ketchup"
+
+/obj/item/storage/box/halloween/edition_19/ketchup/PopulateContents()
+	new /obj/item/clothing/suit/ketchup(src)
+	new /obj/item/clothing/head/ketchup_head(src)
+
+//--Mustard
+/obj/item/clothing/suit/mustard
+	name = "Mustard"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:51]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "A soft plush mustard bottle."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "mustard"
+
+/obj/item/clothing/head/mustard_head
+	name = "Mustard hat"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:58]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+/obj/item/storage/box/halloween/edition_19/mustard
+	theme_name = "2019's Mustard"
+
+/obj/item/storage/box/halloween/edition_19/mustard/PopulateContents()
+	new /obj/item/clothing/suit/mustard(src)
+	new /obj/item/clothing/head/mustard_head(src)
+
+//Angel
+/obj/item/clothing/suit/angel
+	name = "Angel dress"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:75]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "Heavenly dress."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "angel"
+
+/obj/item/clothing/head/angel_halo
+	name = "Angel halo"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:82]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 9]
```diff
@@ -0,0 +1,311 @@
...
+	theme_name = "2019's Angel"
+
+/obj/item/storage/box/halloween/edition_19/angel/PopulateContents()
+	new /obj/item/clothing/suit/angel(src)
+	new /obj/item/clothing/head/angel_halo(src)
+
+//--Devil
+/obj/item/clothing/suit/devil
+	name = "Devil suit"
+	desc = "The one the only Devil."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:98]: ...and here.
Also description is a bit weird, should be changed.

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 10]
```diff
@@ -0,0 +1,311 @@
...
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "devil"
+
+/obj/item/clothing/head/devil_horns
+	name = "Devil horns"
+	desc = "The one the only Devil."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:105]: same here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 11]
```diff
@@ -0,0 +1,311 @@
...
+/obj/item/storage/box/halloween/edition_19/devil
+	theme_name = "2019's Devil"
+
+/obj/item/storage/box/halloween/edition_19/devil/PopulateContents()
+	new /obj/item/clothing/suit/devil(src)
+	new /obj/item/clothing/head/devil_horns(src)
+
+//Cats
+/obj/item/clothing/suit/cat
+	name = "Cat suit"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:120]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 12]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "You feel like you can slink everywhere now."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "cat"
+
+/obj/item/clothing/head/cat_head
+	name = "Cat hat"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:127]: and here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 13]
```diff
@@ -0,0 +1,311 @@
...
+	theme_name = "2019's Cat"
+
+/obj/item/storage/box/halloween/edition_19/cat/PopulateContents()
+	new /obj/item/clothing/suit/cat(src)
+	new /obj/item/clothing/head/cat_head(src)
+
+//Pumpkin
+/obj/item/clothing/suit/space/hardsuit/toy/pumpkin
+	name = "Pumpkin"
+	desc = "The insides of a pumpkin a awkwardly warm and slimy."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:143]: same here, and description needs to be changed

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 14]
```diff
@@ -0,0 +1,311 @@
...
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "pumpkin"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/toy/pumpkin_top
+
+/obj/item/clothing/head/helmet/space/hardsuit/toy/pumpkin_top
+	name = "Pumpkin top"
+	desc = "The top of a pumpkin on the top of your head."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:151]: same here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 15]
```diff
@@ -0,0 +1,311 @@
...
+
+/obj/item/storage/box/halloween/edition_19/pumpkin
+	theme_name = "2019's Pumpkin"
+
+/obj/item/storage/box/halloween/edition_19/pumpkin/PopulateContents()
+	new /obj/item/clothing/suit/space/hardsuit/toy/pumpkin(src)
+
+//Skeleton
+/obj/item/clothing/suit/skeleton
+	name = "Skeleton"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:164]: lowercase, also should have a better name

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 16]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "You are now Mr.Bones."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "skeleton"
+
+/obj/item/clothing/head/skull
+	name = "Skull"
+	desc = "A skull on your face."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:172]: same as last comment, also description is weird

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 17]
```diff
@@ -0,0 +1,311 @@
...
+
+
+/obj/item/storage/box/halloween/edition_19/skeleton/PopulateContents()
+	new /obj/item/clothing/suit/skeleton(src)
+	new /obj/item/clothing/head/skull(src)
+
+//Spider
+/obj/item/clothing/suit/spider
+	name = "Spider"
+	desc = "An unwieldy set of 8 limbs"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:190]: name, and description has no punctuation.

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 18]
```diff
@@ -0,0 +1,311 @@
...
+/obj/item/storage/box/halloween/edition_19/spider
+	theme_name = "2019's Spider"
+
+
+/obj/item/storage/box/halloween/edition_19/spider/PopulateContents()
+	new /obj/item/clothing/suit/spider(src)
+
+//Witch
+/obj/item/clothing/suit/witch
+	name = "witch"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:204]: name shouldn't be just "witch"

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 19]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "You've become the Wicked Witch of the West."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "witch"
+
+/obj/item/clothing/head/witch_hat
+	name = "Witch Head"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:211]: lowercase, also weird name

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 20]
```diff
@@ -0,0 +1,311 @@
...
+	theme_name = "2019's Witch"
+
+
+/obj/item/storage/box/halloween/edition_19/witch/PopulateContents()
+	new /obj/item/clothing/suit/witch(src)
+	new /obj/item/clothing/head/witch_hat(src)
+
+//Sailor Moon
+/obj/item/clothing/suit/sailor_moon
+	name = "Sailor moon leotard"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:229]: lowercase, also maybe more subtle

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 21]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "A magical girl leotard."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "sailormoon_suit"
+
+/obj/item/clothing/head/hat/moon_wig
+	name = "Sailor moon wig"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:236]: lowercase and subtleness

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 22]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "Hair that looks like a pair of meatballs and spaghetti."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "sailormoon_hair"
+	dynamic_hair_suffix = ""
+	flags_inv = HIDEHAIR|HIDEEARS
+
+/obj/item/clothing/gloves/moon_gloves
+	name = "Sailor moon golves"
+	desc = "In the name I will Punish you!"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:246]: Grammar also name

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 23]
```diff
@@ -0,0 +1,311 @@
...
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "sailormoon_gloves"
+
+/obj/item/clothing/shoes/moon_boots
+	name = "Sailor moon boots"
+	desc = "Boots for fighting evil by moonlight."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:253]: Same here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 24]
```diff
@@ -0,0 +1,311 @@
...
+
+/obj/item/storage/box/halloween/edition_19/sailor_moon/PopulateContents()
+	new /obj/item/clothing/suit/sailor_moon(src)
+	new /obj/item/clothing/head/hat/moon_wig(src)
+	new /obj/item/clothing/gloves/moon_gloves(src)
+	new /obj/item/clothing/shoes/moon_boots(src)
+
+//--Tuxedomask
+/obj/item/clothing/suit/tuxedo
+	name = "Tuxedo"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:269]: lowercase and name

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 25]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "A dapper tux."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "tuxedomask_suit"
+
+/obj/item/clothing/head/hat/tuxedo_hat
+	name = "Top hat"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:276]: same here

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 26]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "A dapper top hat."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "tuxedomask_hat"
+
+/obj/item/clothing/gloves/tuxedo_gloves
+	name = "Tuxedo gloves"
+	desc = "Don't prick your finger on that rose"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:284]: lowercase and change description

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 27]
```diff
@@ -0,0 +1,311 @@
...
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "tuxedomask_gloves"
+
+/obj/item/clothing/shoes/tuxedo_shoes
+	name = "Dress shoes"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:290]: lowercase

File: code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm [hunk 28]
```diff
@@ -0,0 +1,311 @@
...
+	desc = "Dress shoes for a dapper tux."
+	icon = 'icons/fulpicons/halloween_costumes/2019/jo_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/jo_costumes.dmi'
+	icon_state = "tuxedomask_shoes"
+
+/obj/item/clothing/mask/tuxedo_mask
+	name = "Dress shoes"
+	desc = "Dress shoes for a dapper tux."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/jo_costumes.dm:298]: epic copypaste fail

File: code/FulpstationCode/halloween_costumes/2019/joseph_joestar.dm [hunk 29]
```diff
@@ -0,0 +1,48 @@
...
+/obj/item/clothing/shoes/tricksters_boots
+	name = "trickster's boots"
+	desc = "These help you Stand."
+	icon = 'icons/fulpicons/halloween_costumes/2019/costumes_icon.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2019/costumes_worn.dmi'
+	icon_state = "joseph_boots"
+
+/obj/item/clothing/gloves/tricksters_gloves
+	name = "trickster's gloves"
+	desc = "Fingerless, to let you better pull tricks out of nowhere."
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/joseph_joestar.dm:35]: description's a bit weird

File: code/FulpstationCode/halloween_costumes/2019/zombie_rider.dm [hunk 30]
```diff
@@ -0,0 +1,16 @@
+/obj/item/clothing/suit/zombie_rider //Spookz
+	name = "Zombie Rider"
+	desc = "Tired of walking? Have your friendly slave zombie waddle you around!"
+	fulp_item = TRUE //I don't know if this is necessary but WHATEVER
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2019/zombie_rider.dm:4]: name also fulp item does nothing

File: code/FulpstationCode/halloween_costumes/2020/2020_event.dm [hunk 31]
```diff
@@ -0,0 +1,137 @@
...
+
+/datum/round_event/halloween_costumes/announce()
+	priority_announce("To improve morale, we've utilized our Bluespace Wardrobe technology to send bulk halloween costumes over. Enjoy.", "Its Spooky Time")
+
+GLOBAL_LIST_EMPTY(possible_h_gifts)
+
+//--- Gift and box with costumes
+/obj/item/halloween_gift
+	name = "halloween package"
+	desc = "It looks like a box wrapped in some spooky paper"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/2020_event.dm:27]: punctuation

File: code/FulpstationCode/halloween_costumes/2020/columbia.dm [hunk 32]
```diff
@@ -0,0 +1,52 @@
+//--Joyce's costume
+/obj/item/clothing/under/columbia
+	name = "Columbia's suit"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/columbia.dm:3]: lowercase

File: code/FulpstationCode/halloween_costumes/2020/columbia.dm [hunk 33]
```diff
@@ -0,0 +1,52 @@
...
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/columbia_worn.dmi'
+	icon_state = "columbia"
+	body_parts_covered = CHEST|GROIN
+	fitted = NO_FEMALE_UNIFORM
+	has_sensor = HAS_SENSORS
+	random_sensor = TRUE
+	can_adjust = FALSE
+
+/obj/item/clothing/head/columbia
+	name = "Columbia's hat"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/columbia.dm:15]: lowercase

File: code/FulpstationCode/halloween_costumes/2020/columbia.dm [hunk 34]
```diff
@@ -0,0 +1,52 @@
...
+	desc = "You're into a time slip..."
+	icon = 'icons/fulpicons/halloween_costumes/2020/columbia_item.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/columbia_worn.dmi'
+	icon_state = "columbia_hat"
+
+/obj/item/clothing/suit/columbia
+	name = "Columbia's jacket"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/columbia.dm:22]: lowercase

File: code/FulpstationCode/halloween_costumes/2020/columbia.dm [hunk 35]
```diff
@@ -0,0 +1,52 @@
...
+	desc = "And nothing can ever be the same."
+	icon = 'icons/fulpicons/halloween_costumes/2020/columbia_item.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/columbia_worn.dmi'
+	icon_state = "columbia_jacket"
+	allowed = list(/obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+
+/obj/item/clothing/neck/columbia
+	name = "Columbia's bowtie"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/columbia.dm:30]: same here

File: code/FulpstationCode/halloween_costumes/2020/columbia.dm [hunk 36]
```diff
@@ -0,0 +1,52 @@
...
+	desc = "Let's do the time warp again!"
+	icon = 'icons/fulpicons/halloween_costumes/2020/columbia_item.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/columbia_worn.dmi'
+	icon_state = "columbia_bow"
+
+/obj/item/clothing/shoes/columbia
+	name = "Columbia's shoes"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/columbia.dm:37]: and here

File: code/FulpstationCode/halloween_costumes/2020/daft_frank.dm [hunk 37]
```diff
@@ -0,0 +1,80 @@
...
+	icon_state = "hardhat0_thomas"
+	brightness_on = 1
+	on = FALSE
+	hat_type = "thomas"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	resistance_flags = null
+	flags_inv = HIDEHAIR|HIDEFACIALHAIR
+	clothing_flags = SNUG_FIT
+	flags_cover = HEADCOVERSEYES
+	dynamic_hair_suffix = ""
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/daft_frank.dm:34]: names should be subtle, also why are they hardhats, this doesn't overwrite the action types either

File: code/FulpstationCode/halloween_costumes/2020/daft_frank.dm [hunk 38]
```diff
@@ -0,0 +1,80 @@
...
+
+/obj/item/clothing/under/sparky
+	name = "The suit"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/daft_frank.dm:37]: lowercase

File: code/FulpstationCode/halloween_costumes/2020/daft_frank.dm [hunk 39]
```diff
@@ -0,0 +1,80 @@
...
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_worn.dmi'
+	icon_state = "the_suit"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	fitted = FEMALE_UNIFORM_FULL
+	has_sensor = HAS_SENSORS
+	random_sensor = TRUE
+	can_adjust = FALSE
+
+/obj/item/clothing/gloves/daft_golden
+	name = "insert name here"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/daft_frank.dm:49]: name

File: code/FulpstationCode/halloween_costumes/2020/daft_frank.dm [hunk 40]
```diff
@@ -0,0 +1,80 @@
...
+	desc = "A pair of sparkly golden gloves."
+	icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_item.dmi'
+	worn_icon = 'icons/fulpicons/halloween_costumes/2020/daft_frank_worn.dmi'
+	icon_state = "golden_gloves"
+
+/obj/item/clothing/gloves/daft_silver
+	name = "insert name here"
```
Comment (SgtHunk) [inline code/FulpstationCode/halloween_costumes/2020/daft_frank.dm:56]: name

**Review by SgtHunk — CHANGES_REQUESTED** [review]

**Comment by GuillaumePrata** [comment]
WTF do I still have this open? I blame Mith either way!!1!

**Pull request closed without merging**